### PR TITLE
added Dockerfile Archive project

### PR DESCRIPTION
**Pull request by darkmatter18**

### CODE REVIEW

File: _data/projects/dockerfile-archieve.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+- docker
+- dockerfile
+- container
+- python
+- javascript
+- cloud
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/darkmatter18/dockerfile-archive/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/dockerfile-archieve.yml:16]: There are no listed issues currently with this label, which may mean this won't be listed on the site. Do you plan to add some tasks @darkmatter18 ?

**Comment by AbsolytnaiaLove** [comment]
сб, 8 жовт. 2022, 17:37 користувач Brendan Forster ***@***.***>
пише:

File: _data/projects/dockerfile-archieve.yml (hunk 1 above)
Comment (darkmatter18) [reply _data/projects/dockerfile-archieve.yml:16]: @shiftkey I'll add some tass today

**Comment by AbsolytnaiaLove** [comment]
нд, 9 жовт. 2022, 11:22 користувач Arkadip Bhattacharya <
***@***.***> пише:

File: _data/projects/dockerfile-archieve.yml (hunk 1 above)
Comment (darkmatter18) [reply _data/projects/dockerfile-archieve.yml:16]: @shiftkey I added some tasks. Will add some more

**Review by shiftkey — APPROVED** [review]

**Pull request merged**